lib/helper.coffee

```diff
@@ -0,0 +1,91 @@
+ChildProcess = require 'child_process'
+Path = require 'path'
+FS = require 'fs'
+{XRegExp} = require 'xregexp'
+
+class Helpers
+  @findFile: (directory, fileName) ->
+    climb = directory.split(Path.sep)
+    while climb.length
+      target = Path.join(climb.join(Path.sep), fileName)
+      if FS.existsSync(target)
+        return target
+      climb.pop()
+  @exec: (command, cwd, stdin) ->
```

Comment:
We should have command and arguments so that we don't run into problems where paths with spaces break arguments into pieces